backend/models/messageModel.js

```diff
@@ -6,9 +6,20 @@ const messageSchema = mongoose.Schema(
     content: { type: String, trim: true },
     chat: { type: mongoose.Schema.Types.ObjectId, ref: "Chat" },
     readBy: [{ type: mongoose.Schema.Types.ObjectId, ref: "User" }],
+    reactions: [
+      {
+        user: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
+        reaction: { 
+          type: String, 
+          enum: ['👍', '❤️', '😂', '😢', '🎉'], // Updated to match emojis used
```

Comment:
Can we not use enum here? In future if we want to add a new reaction, then we have to update this enum